bin/katello-certs-check

```diff
@@ -211,6 +211,38 @@ function check-cert-usage-key-encipherment () {
     fi
 }
 
+function check-shortname () {
+    printf "Checking for use of shortname as CN"
+
+    SUBJECT_CN=$(openssl x509 -in $CERT_FILE -noout -subject -nameopt multiline|grep commonName|cut -d '=' -f 2|tr -d ' ')
+    if [[ $SUBJECT_CN != *"."* ]]; then
+        error 1 "The $(basename $CERT_FILE) is using a shortname for Common Name (CN) and cannot be used with $PROJECT.\n"
+    fi
+
+    DNS_LIST=$(openssl x509 -noout -text -in $CERT_FILE | grep DNS:)
+    if [[ $? == "0" ]]; then
+        if [[ "${#DNS_LIST[@]}" == 1 ]]; then
```

Comment:
Why is it needed to special case this (other than the error message)? Doesn't the `-gt 1` code handle this already?